<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathSettingsDialog</class>
 <widget class="QDialog" name="PathSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>路径设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>请配置游戏相关路径</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14pt; font-weight: bold; margin: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pathGroupBox">
     <property name="title">
      <string>路径配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="steamPathLabel">
        <property name="text">
         <string>Steam 安装路径:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="steamPathEdit">
        <property name="placeholderText">
         <string>例如: C:/Program Files (x86)/Steam</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="browseSteamButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="gamePathLabel">
        <property name="text">
         <string>游戏安装路径:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="gamePathEdit">
        <property name="placeholderText">
         <string>例如: C:/Program Files (x86)/Steam/steamapps/common/RimWorld</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browseGameButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="savePathLabel">
        <property name="text">
         <string>用户存档路径:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="savePathEdit">
        <property name="placeholderText">
         <string>例如: C:/Users/YourName/AppData/LocalLow/Ludeon Studios/RimWorld by Ludeon Studios</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="browseSaveButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hintLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#666666;&quot;&gt;提示：用户存档路径已使用默认值填充，通常无需修改。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PathSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PathSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
